addons/redirect-mobile-forums/addon.json

```diff
@@ -10,11 +10,11 @@
   "userscripts": [
     {
       "url": "userscript.js",
-      "matches": ["https://scratch.mit.edu/discuss/m/*"],
+      "matches": ["https://scratch.mit.edu/discuss/m/*", "https://scratch.mit.edu/404"],
```

Comment:
If the userscript runs in 404, it needs to be changed so that it only does stuff if the URL path starts with `/discuss/m/` or is exactly `/discuss/m`